README.md

```diff
@@ -1,10 +1,177 @@
-# SigLens Docs
-### Installation
-```
-$ npm install
-```
+# Welcome to SigLens Documentation!
+
+<p align="center">
+<img src="https://github.com/siglens/siglens/assets/604069/7dab105b-2102-4a32-85c7-02fbb4604217" width="300">
+</p>
+
+
+Welcome to the official documentation repository for SigLens, the cutting-edge, open-source alternative to Splunk/DataDog. Our mission is to revolutionize the observability landscape by providing a solution that is not only 100x more efficient than Splunk but also significantly reduces your observability costs—by up to 90%.
+
+This repository is the source of the [Official SigLens documentation](https://siglens.github.io/siglens-docs). We appreciate your interest in contributing and welcome your participation in making this resource even better. If you have any questions, please reach out to us on [Slack](https://www.siglens.com/slack.html).
+
+# Table of Contents
+
+- [Introduction](#introduction)
+- [Getting Started](#getting-started)
+- [Documentation Workflow](#documentation-workflow)
+  - [Markdown structure](#markdown-structure)
+- [How to Contribute](#how-to-contribute)
+  - [Content Guidelines](#content-guidelines)
+  - [First-Time Contributors](#first-time-contributors)
+- [Community and Support](#community-and-support)
+- [License](#license)
+
+## Introduction
+
+SigLens is engineered for excellence in managing massive volumes of data, from terabytes to petabytes, without compromising on speed or efficiency. At the heart of SigLens is our proprietary microindexing technology, which ensures that micro indices are only 1/100th the size of traditional indices, resulting in smaller index sizes and faster ingestion speeds. Moreover, SigLens utilizes dynamic columnar compression to reduce on-disk storage size to often just 1% of the incoming volume. This innovation not only enhances query times but also substantially lowers storage and compute requirements.
+
+## Getting Started
+
+Before you start contributing to SigLens Documentation, here are a few things you should know:
+
+- **Markdown**: Our documentation is written in Markdown. Familiarity with Markdown syntax is helpful but not required.
+- **Git and GitHub**: Contributions are made via GitHub. A basic understanding of how to fork a repository, create branches, and submit pull requests is necessary.
+
+**SigLens Platform**: Having a basic understanding of what SigLens offers will help you write better documentation. We encourage you to try out SigLens using 
```

Comment:
Is this section about getting started with using siglens or getting started contributing? I think just having this be getting started using siglens makes more sense since the previous paragraph was just very basic information about siglens, so the reader probably isn't familiar with the code and would want to try it out before they start contributing. So I think you can move lines 30-33 down to the contributing section you have below, and then on line 35 you can remove some of the info about writing documentation (so maybe just have the second sentence, or do what sounds right to you)